catroid/src/main/java/org/catrobat/catroid/ui/MainMenuActivity.java

```diff
@@ -97,6 +99,26 @@ public void handleAgreedToPrivacyPolicyButton(View view) {
 		loadContent();
 	}
 
+	private boolean hasUserUpdatedPocketCode() {
+		try {
+			long versionCode =
+					getPackageManager().getPackageInfo(getPackageName(), 0).getLongVersionCode();
```

Comment:
just had a look at jenkins and the documentation, you cant do this, getLongVersionCode was just recently added in API28